lib/src/slider/slider.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+import 'package:modular_ui/src/slider/components/custom_slider_thumb_shape.dart';
+
+class MUISlider extends StatefulWidget {
+  const MUISlider(
+      {Key? key,
+      this.height = 5,
+      this.inactiveColor = const Color(0xffECEFF1),
+      this.activeColor = Colors.black,
+      this.thumbColor = Colors.white,
+      required this.onChanged,
+      required this.value,
+      required this.min,
+      required this.max,
+      this.onChangeStart,
+      this.onChangeEnd})
+      : assert(min <= max),
+        assert(value >= min && value <= max,
+            'Value $value is not between minimum $min and maximum $max'),
+        super(key: key);
+
+  /// Height for Slider ; default: 5
+  final double? height;
+
+  /// Inactive Color for Slider ; default: Colors.grey
+  final Color? inactiveColor;
+
+  /// Active Color for Slider ; default: Colors.black
+  final Color? activeColor;
+
+  /// Thumb Color for Slider ; default: Colors.black
+  final Color? thumbColor;
+
+  /// OnChanged for Slider
+  final void Function(double)? onChanged;
```

Comment:
Just keep it non-nullable because the value of the slider is anyway required by the user. so better to hand over the control to the end user.

Remove the nullable here and keep the argument required (no change in the constructor).

Reply:
ok got it.. done